### ISSUE TITLE
Small objects using THREE.TransformControls scale too slow

### ISSUE THREAD

**Issue by nicolettiant89** [mrdoob/three.js#8891]
##### Description of the problem 
Hi, I noticed that when I try to scale a small object using THREE.TransformControls it scales too slow. The problem is that TransformControls scales the distance of the mouse movement by a seemingly arbitrary amount (1/50) within the onMouseMove function:

`  } else if (_mode == "scale") {

            point.sub(offset);
            point.multiply(parentScale);

            if (scope.space == "local") {

                if (scope.axis == "XYZ") {

                    scale = 1 + ((point.y) / 50);

                    scope.object.scale.x = oldScale.x * scale;
                    scope.object.scale.y = oldScale.y * scale;
                    scope.object.scale.z = oldScale.z * scale;

                } else {

                    point.applyMatrix4(tempMatrix.getInverse(worldRotationMatrix));

                    if (scope.axis == "X") scope.object.scale.x = oldScale.x * (1 + point.x / 50);
                    if (scope.axis == "Y") scope.object.scale.y = oldScale.y * (1 + point.y / 50);
                    if (scope.axis == "Z") scope.object.scale.z = oldScale.z * (1 + point.z / 50);

                }

            }

        } `
This parameter (50) effectively controls the "speed" at which the object is scaled (compared to how far the mouse is dragged). 
So can I suggest to include in TransformControls.js file a new function like this?:

`this.setScaleSpeed = function ( speed ) {
                  scaleSpeed = speed;
               };`

where the local variable "speed" could be for example the bounding sphere radius of a mesh I need to scale; and "scaleSpeed" could be a global variable, included into the file like "var scaleSpeed=50" and used in the previous method in such a way:

` } else if (_mode == "scale") {

            point.sub(offset);
            point.multiply(parentScale);

            if (scope.space == "local") {

                if (scope.axis == "XYZ") {

                    scale = 1 + ((point.y) / scaleSpeed);

                    scope.object.scale.x = oldScale.x * scale;
                    scope.object.scale.y = oldScale.y * scale;
                    scope.object.scale.z = oldScale.z * scale;

                } else {

                    point.applyMatrix4(tempMatrix.getInverse(worldRotationMatrix));

                    if (scope.axis == "X") scope.object.scale.x = oldScale.x * (1 + point.x / scaleSpeed);
                    if (scope.axis == "Y") scope.object.scale.y = oldScale.y * (1 + point.y / scaleSpeed);
                    if (scope.axis == "Z") scope.object.scale.z = oldScale.z * (1 + point.z / scaleSpeed);

                }

            }

        }`
In other words, "scaleSpeeds" would sostitute the scalar value 50.
It would resolve this issue, and my problem, very easily. What do you think?
...

##### Three.js version

- [ ] Dev
- [ ] r76
- [ x] r74

##### Browser

- [x] All of them
- [] Chrome
- [ ] Firefox
- [ ] Internet Explorer

##### OS

- [x] All of them
- [ ] Windows
- [ ] Linux
- [ ] Android
- [ ] IOS

**Comment by WestLangley** [comment]
That scaling issue was [fixed](https://github.com/mrdoob/three.js/commit/3eb8990e7d6e1aed86415c5708d830f11d643cb6) in r.76.

**Comment by nicolettiant89** [comment]
Thanks a lot.
